### DISCUSSION TITLE
Message validation

### DISCUSSION THREAD

**Discussion by eaziweb95** [filamentphp/filament#2796] [Help] [answered]
When i validate a form i have the message **validation.exists** under a select field, what does it means ? is it possible to translate this message ? in which file ?

**Comment by eaziweb95** [comment]
![validation](https://user-images.githubusercontent.com/58070424/174096676-ce5b559c-4580-43ac-b623-728f831a501b.jpg)

**Comment by danharrin** [comment]
It's in the default Laravel validation file... set your fallback_locale in config/app to `en`

**Comment by eaziweb95** [comment, marked as answer]
Thanks !
